apps/system/js/app_chrome.js

```diff
@@ -137,11 +133,15 @@
   AppChrome.prototype.view = function an_view() {
     var className = this.CLASS_NAME + this.instanceID;
 
-    return `<div class="chrome" id="${className}">
+    return `<div class="chrome chrome-plain" id="${className}">
             <gaia-progress></gaia-progress>
             <section role="region" class="bar">
-              <gaia-header action="close">
+              <gaia-header action="close" class='js-chrome-ssl-information'>
+                <div class="chrome-ssl-indicator chrome-ssl-indicator-ltr">
+                </div>
                 <h1 class="title"></h1>
+                <div class="chrome-ssl-indicator chrome-ssl-indicator-rtl">
```

Comment:
we have 2 elements with the "chrome-ssl-indicator" class because we need to have it at the left in LTR and at the right at RTL. We'll be able to remove this when gaia-header is not forced LTR anymore.